app/index.html

```diff
@@ -61,10 +61,9 @@
 <script src="js/inc/shim/Base64.js" type="text/javascript"></script>
 <script src="js/inc/shim/Base64binary.js" type="text/javascript"></script>
 <script src="js/inc/shim/WebAudioAPI.js" type="text/javascript"></script>
-<script src="js/bower/requirejs/require.js"></script>
 <script src="js/inc/WebMIDIAPI.min.js"></script>
-<!-- build:js({.tmp}) js/main.js -->
-<script src="js/init.js"></script>
-<!-- endbuild -->
+
+<script data-main="js/init.js" src="js/bower/requirejs/require.js"></script>
```

Comment:
It's *require.js*-only attribute =) one of the possible ways to specify main script entry point. And here it's necessary cause grunt-requirejs **almondify-feature** uses it to find and replace script tag.